test/attr/attr_autoclosure.swift

```diff
@@ -234,3 +234,9 @@ func rdar_30906031(in arr: [Int], fn: @autoclosure () -> Int) -> Bool {
     arr.lazy.filter { $0 >= escapableF() }.isEmpty
   }
 }
+
+// SR-2688
+class Foo {
+  typealias FooClosure = () -> String
```

Comment:
Could you add a typealias that has a non-`Void` argument type to make sure we diagnose the typealias properly?

Reply:
Done! Can you invoke the CI for me?